content/03-reference/01-tools-and-interfaces/02-prisma-client/030-crud.mdx

```diff
@@ -654,6 +660,8 @@ const user = await prisma.user.delete({
 
 The `updateMany` query updates a batch of existing database records in bulk and returns the number of deleted records. You can [filter](filtering) the list of records to be updated.
 
+To perform arithmeic operations (add, subtract, multiply), use [atomic updates](#atomic-operations-on-update) to prevent race conditions.
```

Comment:
```suggestion
To perform arithmeic operations (add, subtract, multiply, divide), use [atomic updates](#atomic-operations-on-update) to prevent race conditions.
```